app/graphql/types/query_types/survey_poll_type.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Types
+  class QueryTypes::SurveyPollType < Types::BaseObject
+    field :id, ID, null: true
+    field :title, QueryTypes::I18nTextType, null: true
+    field :question_title, QueryTypes::I18nTextType, null: true
+    field :description, QueryTypes::I18nTextType, null: true
+    field :date, QueryTypes::DateType, null: true
+    field :response_options, [QueryTypes::SurveyResponseOptionsType], null: true
+    field :updated_at, String, null: true
+    field :created_at, String, null: true
```

Comment:
what does the `null: true` do?

Reply:
it is allowed to be `null`

Reply:
good question. currently every field of every type has this set. i think it defines optional fields, that could be null. i will have a short research.
it does not makes sense at all like it is at the moment. some fields should not have that, as they are required and never null.
needs to be updated across all types somewhen. we could directly correct this here for surveys from the beginning.

Reply:
![Bildschirmfoto 2021-06-16 um 18 08 37](https://user-images.githubusercontent.com/1942953/122254901-e45f4b00-cecd-11eb-8c2a-b30ce128ad9a.png)
https://graphql-ruby.org/fields/introduction.html